pkg/serverless/invocationlifecycle/trace.go

```diff
@@ -63,6 +63,9 @@ func (lp *LifecycleProcessor) startExecutionSpan(event interface{}, rawPayload [
 			inferredSpan.Span.TraceID = traceContext.TraceID
 			inferredSpan.Span.ParentID = traceContext.ParentID
 		}
+		if traceContext.TraceIDUpper64Hex != "" {
+			lp.requestHandler.SetMetaTag(ddTraceIDUpper64BitsHeader, traceContext.TraceIDUpper64Hex)
```

Comment:
Am I understanding this correctly, the upper 64 bits are set as a span tag? They are not included in any in/outbound trace context headers?

Reply:
Okay, this is an actual concern that I believe could become a actual problem.

Since the `requestHandler` is _not_ reset on each invocation, it's feasible that one invocation has the upper 64 and then one later on does not. The later one would then still get the meta tag from the previous invocation. That's a problem.

Reply:
Hm. That is a concern. A lambda function could be invoked by a Step Function and then a non-Step-Function-thing. That would be rare, but not impossible. Would it be reasonable to remove that Meta tag in OnInvokeEnd?

Reply:
I've removed the meta tag here; instead I'm passing the upper 64 in the response that goes back to the tracer.